operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -155,17 +157,17 @@ or [Upgrade CSM and additional products with IUF](../workflows/upgrade_csm_and_a
 
 1. (`ncn-mw#`) Determine the xnames for the NCNs which are being upgraded. These will be used in the next step.
 
-    - Get a comma-separated list of all storage NCN xnames:
+    - Get the xname for `ncn-m001`:
```

Comment:
Don't we have different images for ncn-m001 and the storage nodes?  Won't these instructions set them all to the same value?

Reply:
Yeah, your change helps a lot, thanks!  A title of ```Set NCN boot image for NCN master or NCN storage nodes``` (change the ```and``` to an ```or```) might help too - your call on that one.

Reply:
Sounds good. I changed the 'and' to 'or'